vfo/internal_database_functions.py

```diff
@@ -295,7 +295,7 @@ def _readNodeDispData(ModelName,LoadCaseName):
 	LoadCaseDir = os.path.join(ODBdir, LoadCaseName)
 	
 	# Get number of nodes in the model to set a node displacement array
-	nodes,elements = _readNodesandElements(ModelName)
+	nodes,elements, eleclass = _readNodesandElements(ModelName)
```

Comment:
Change it to 'eleClassTags' to keep consistent with the other changes you have proposed.